README.md

```diff
@@ -215,11 +215,13 @@ so for the time being our tests run using both.
 To run all integration tests from the command-line:
 
 ```bash
-SMOKE_USERNAME=username SMOKE_PASSWORD=password ROOT_URL=https://scratch.mit.edu npm run test:integration
+SMOKE_USERNAME=username SMOKE_PASSWORD=password ROOT_URL=https://scratch.mit.edu PROJECT_ID=id# npm run test:integration
```

Comment:
I think we should keep this copy-pasteable

```suggestion
SMOKE_USERNAME=username SMOKE_PASSWORD=password ROOT_URL=https://scratch.mit.edu TEST_PROJECT_ID=1 npm run test:integration
```

Reply:
Sounds good